Source/ASTextNode.mm

```diff
@@ -396,7 +396,7 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   // Add the constrained size back textContainerInset
   size.width += (_textContainerInset.left + _textContainerInset.right);
   size.height += (_textContainerInset.top + _textContainerInset.bottom);
-  
```

Comment:
@wsdwsd0829 you can delete these whitespace changes, and also revert the diffs to Kittens / CatDealsCollectionView / ASDKgram.

Will catch up with you offline to talk about potentially putting the new view controller in examples_extra/TextStressTest; this would reduce some of the other boilerplate and might help us centralize our reproduction cases for text drawing issues.